tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
```

Comment:
i understand we need the __init__.py file - but are the contents required or used anywhere (like does this hook in via our CI in GitHub?)

Reply:
Currently, we do require the content so that we can reference and import the classes from the top level caldera repo. I pushed a change to insert this path as a string, because sys.path only accepts strings. [https://docs.python.org/3/library/sys.html](url)

Reply:
It's not needed just yet, but will be when we figure out how we want to test this (and other) plugins -- since the plugins all reference/import core code.

Reply:
^^ sounds good